src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Perhaps can include some header comments for the interface to explain what should it be use for etc.